### ISSUE TITLE
raining couldn 't start

### ISSUE THREAD

**Issue by zwyking** [vacancy/Synchronized-BatchNorm-PyTorch#45]
Hi,

I tried to used it as

model = Model(...)
model = nn.DataParallel(model)
model = convert_model(model)
model = model.cuda()

However, it stucked and training couldn 't start.
Have you seem similar problems before ?

**Comment by vacancy** [comment]
See also https://github.com/vacancy/Synchronized-BatchNorm-PyTorch/issues/44#issuecomment-815135207

**Comment by zwyking** [comment]
> See also [#44 (comment)](https://github.com/vacancy/Synchronized-BatchNorm-PyTorch/issues/44#issuecomment-815135207)

ok, thanks a lot
